File: smartwire-backend_server/src/main/java/jpcompany/smartwire/security/jwt/handler/dto/JwtMemberDto.java
package jpcompany.smartwire.security.jwt.handler.dto;

import lombok.Getter;
import lombok.ToString;

@Getter @ToString
public class JwtMemberDto {
    private Integer id;
    private String loginId;
    private String companyName;
    private String email;
    private String phoneNumber;
    private String role;
}

File: smartwire-window_desktop-1.2/dto/JwtMemberDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace smartwire_window_desktop.dto
{
    internal class JwtMemberDto
    {
        public int Id { get; set; }
        public string LoginId { get; set; }
        public string CompanyName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string Role { get; set; }
    }
}
